kubernetes-model/cmd/generate/generate.go

```diff
@@ -311,6 +312,7 @@ func main() {
     {"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1", "", "io.fabric8.kubernetes.api.model.apiextensions", "kubernetes_apiextensions_"},
     {"k8s.io/apimachinery/pkg/apis/meta/v1", "", "io.fabric8.kubernetes.api.model", "kubernetes_apimachinery_"},
     {"k8s.io/api/networking/v1", "networking.k8s.io", "io.fabric8.kubernetes.api.model.networking", "kubernetes_networking_"},
```

Comment:
Do we have test coverage for Ingress under networking.k8s.io/v1 as well? 
I'm not sure if we need it or not but just asking.

Reply:
I think there is `IngressTest` already